### ISSUE TITLE
Post images without alt_text

### ISSUE THREAD

**Issue by eliocamp** [schochastics/rtoot#129]
I'm trying to post an image and I get an error if I don't add an alt text. While using alt text should be encouraged, it is not a requirement of the API.

**Comment by schochastics** [comment]
This functionality is modeled after rtweet (https://github.com/ropensci/rtweet/blob/master/R/post-tweet.R#L269) which has the same requirement. I would want to keep it this way. A workaround should be to supply an empty string

**Comment by eliocamp** [comment]
Yeah, I think it's a bad idea there too :P. 

Are we sure that using an empty string is better than no alt text? It seems to me that this would tag the images as having alt text without actually having it.

**Comment by schochastics** [comment]
Just tested it. It doesnt tag the image as having an alt_text

**Comment by eliocamp** [comment]
Ok, then.
